commands.js

```diff
@@ -213,6 +213,41 @@ _.GSV0_FORMAT = {
   GSV0_DWDH     : '\x1d\x76\x30\x03'
 };
 
+_.PDF417_FORMAT = {
```

Comment:
Is the pdf417 format a format that works on printers with all esc/pos specifications?